apps/meteor/app/lib/server/functions/createRoom.ts

```diff
@@ -24,6 +24,7 @@ export const createRoom = async <T extends RoomType>(
 	name: T extends 'd' ? undefined : string,
 	ownerUsername: string | undefined,
 	members: T extends 'd' ? IUser[] : string[] = [],
+	excludeSelf?: boolean,
```

Comment:
Couldn't we here transform all the optional parameters into a object so we don't have to add this `excludeSelf` in all other files?

Reply:
At least move the new param to the existing options object - also add a comment to the type definition to clarify what this param does, as "excludeSelf" by itself doesn't say much.

Reply:
I'll work on it in a separate PR just so that we do not hold this one for the patch release